BcSaaS/Import-BcConfigurationPackage.ps1

```diff
@@ -0,0 +1,161 @@
+﻿<# 
+ .Synopsis
+  Function for importing and applying configuration package to online tenant
+ .Parameter environment
+  Name of the environment inside the tenant in which you want to publish the Per Tenant Extension Apps
+ .Parameter companyName
+  Company Name to import the package to
+ .Parameter packageFile
+  File with a package to import. Package code inside file must be equal to packageCode parameter
+ .Parameter packageCode
+  Package will be created with specified code
+ .Parameter packageName
+  Package will be created with specified name
+ .Parameter packageLang
+  Package will be created with specified language id, en-US (1033) by default.
+ .Parameter importPackage
+  Enable the switch to import the package after upload
+ .Parameter applyPackage
+  Enable the switch to apply the package after import
+ .Parameter useNewLine
+  Add this switch to add a newline to progress indicating periods during wait.
+  Azure DevOps doesn't update logs until a newline is added.
+
+#>
+function Import-BcConfigurationPackage {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $bcAuthContext,
+        [Parameter(Mandatory=$true)]
+        [string] $environment,
+        [Parameter(Mandatory=$false)]
+        [string] $companyName,
+        [Parameter(Mandatory=$true)]
+        $packageFile,

```

Comment:
this is called configPackage in the corresponding function for containers
It would be OK to add an alias called packageFile (to both functions)